src/main/java/io/jenkins/plugins/monitoring/MonitorPortlet.java

```diff
@@ -46,7 +54,9 @@
      *          the app relative icon url depending on ${resURL} (/static/cache-id),
      *          or {@code Optional.empty()}, if a default icon should be added.
      */
-    Optional<String> getIconUrl();
+    public Optional<String> getIconUrl() {
```

Comment:
Another topic for our meeting: does this URL allow SVGs? This might be a direction we are heading to in Jenkins for all icons.

Reply:
Yes, its rendered as an img tag, so all common file formates (depends on browser) are supported:  jpeg, gif, png, apng, svg, bmp, bmp ico, png ico.